api/v1/users/PKPUserHandler.inc.php

```diff
@@ -125,9 +125,11 @@ public function getMany($slimRequest, $response, $args)
             ->searchPhrase($params['searchPhrase'] ?? null)
             ->orderBy($orderBy, $orderDirection, [AppLocale::getLocale(), $request->getSite()->getPrimaryLocale()])
             ->limit($params['count'] ?? null)
-            ->offset($params['offset'] ?? null)
-            ->filterByStatus($params['status'] ?? $collector::STATUS_ALL);
-
+            ->offset($params['offset'] ?? null);
```

Comment:
Just updated this to create a discussion about the mapping (there are other similar cases in this file).
Given the `active` is the same as the `Collector::STATUS_ACTIVE`, I think we can just let it pass through and validate it in a single place.

Reply:
I haven't come to a firm idea on this, but you're right that the use of `$params` and the collector functions/constants currently duplicate some things. I would lean towards letting the collectors implement the model, then draw on that in the handler. Personally I don't like the use of arrays (as we're using `$params` here, and previously used them in `EntityDAO` subclasses) -- their contents can't be typed/controlled it's too easy to make a mistake.

Reply:
I'm also against passing static information through dictionaries... In a next iteration, I think we should get rid of them.
But for now, I'll just revert this change =]